### ISSUE TITLE
Add a new setting to exclude inactive enrolments from  the reports

### ISSUE THREAD

**Issue by dmitriim** [marceloschmitt/moodle-block_analytics_graphs#60]
In our case we don't fully unenrol users from courses, but just suspend them and reactivate later when required. In this case all reports are polluted by data that not relevant as it's mostly suspended users. Also when sending emails, then suspended users also get them which is not relevant to them anymore. 

We would like to propose adding a new global plugin setting "Only active users" - consider only active enrolments in all reports.

**Comment by marceloschmitt** [comment]
Dmitri, this one a had temporarily fixed changing the functions that gets the students. I will analyse your code (most certainly perfect) and I can add. I would like to let the teacher configure if he wants to see suspended students. I was not thinking about a global configuration. What do you think?

**Comment by dmitriim** [comment]
@marceloschmitt I think it's a good idea. We can improve our patch to allow more flexibility. We can add an extra setting for a block so teaches can toggle per block. Also we can have a new global setting that can switch this feature off globally and let admins control it site wise. Thoughts?

**Comment by marceloschmitt** [comment]
I agree with everything. Do you intend to code that? Otherwise, I will do it.

**Comment by dmitriim** [comment]
Hi @marceloschmitt 
Yes. We will update the code of https://github.com/marceloschmitt/moodle-block_analytics_graphs/pull/61 with the changes that we discussed

**Comment by dmitriim** [comment]
@marceloschmitt the code is ready for your review https://github.com/marceloschmitt/moodle-block_analytics_graphs/pull/61 
Please let me know if you have any questions.
